recipes/Orange3-Textable/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "Orange3-Textable" %}
+{% set version = "3.1.11" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Orange3-Textable-{{ version }}.tar.gz
+  sha256: 87b070592d6f36bc31f6ae25812ce1b41d058b2a7bd974ef7c66d71319f8bf3a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >3
+  run:
+    - appdirs
+    - chardet
+    - future
+    - lttl >=2.0.12
+    - orange3 >=3.14.0
+    - pycountry
+    - python >3
+    - setuptools
+    - treetaggerwrapper
+
+test:
+  imports:
+    - _textable
+    - _textable.widgets
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://textable.io
+  summary: Textable add-on for Orange 3 data mining software package.
+  license: GPL-3.0
```

Comment:
It seems like the license is `GPL-3.0-or-later`: https://github.com/axanthos/orange3-textable/blob/44ffdd36640ba5e92f8a026794bc4583d31cf3a6/setup.py#L11 Do you know which one it is or can you ask upstream about it?

Reply:
I wrote to the repo owner who told me to use the or-later